tools/minerva/.shed.yml

```diff
@@ -0,0 +1,46 @@
+categories:
+  - Pathway Analysis
```

Comment:
also update this category to an existing toolshed category

and probably not make this one a suite either